content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -69,84 +68,71 @@ In this example:
   * Create one container and name it `nginx` using the `name` field.
   * Open port `80` so that the container can send and accept traffic.
 
-To create this Deployment, run the following command:
-
-```shell
-kubectl apply -f https://k8s.io/examples/controllers/nginx-deployment.yaml
-```
-
-{{< note >}}
-You may specify the `--record` flag to write the command executed in the resource annotation `kubernetes.io/change-cause`. It is useful for future introspection, for example to see the commands executed in each Deployment revision.
-{{< /note >}}
-
-Next, run `kubectl get deployments`. The output is similar to the following:
-
-```shell
-NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-nginx-deployment   3         0         0            0           1s
-```
-
-When you inspect the Deployments in your cluster, the following fields are displayed:
-
-* `NAME` lists the names of the Deployments in the cluster.
-* `DESIRED` displays the desired number of _replicas_ of the application, which
-   you define when you create the Deployment. This is the _desired state_.
-* `CURRENT` displays how many replicas are currently running.
-* `UP-TO-DATE` displays the number of replicas that have been updated to achieve
-   the desired state.
-* `AVAILABLE` displays how many replicas of the application are available to
-   your users.
-* `AGE` displays the amount of time that the application has been running.
-
-Notice how the values in each field correspond to the values in the Deployment specification:
-
-* The number of desired replicas is 3 according to `.spec.replicas` field.
-* The number of current replicas is 0 according to the `.status.replicas` field.
-* The number of up-to-date replicas is 0 according to the `.status.updatedReplicas` field.
-* The number of available replicas is 0 according to the `.status.availableReplicas` field.
-
-To see the Deployment rollout status, run `kubectl rollout status deployment.v1.apps/nginx-deployment`. This command returns the following output:
-
-```shell
-Waiting for rollout to finish: 2 out of 3 new replicas have been updated...
-deployment.apps/nginx-deployment successfully rolled out
-```
-
-Run the `kubectl get deployments` again a few seconds later:
-
-```shell
-NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-nginx-deployment   3         3         3            3           18s
-```
-
-Notice that the Deployment has created all three replicas, and all replicas are up-to-date (they contain the
-latest Pod template) and available (the Pod status is Ready for at least the value of the Deployment's `.spec.minReadySeconds` field).
-
-To see the ReplicaSet (`rs`) created by the deployment, run `kubectl get rs`:
+  Follow the steps given below create the above Deployment:
+
+  Before you begin, make sure your Kubernetes cluster is up and running.
```

Comment:
This reminds me of the `{{% capture prerequisites %}}` at the start of a Task page.

Maybe some of this page works better as a Task?